src/Symfony/Component/Messenger/Tests/Transport/RedisExt/RedisExtIntegrationTest.php

```diff
@@ -104,4 +102,77 @@ public function testConnectionSendDelayedMessagesWithSameContent()
         $this->assertEquals($body, $encoded['body']);
         $this->assertEquals($headers, $encoded['headers']);
     }
+
+    public function testConnectionBelowRedeliverTimeout()
+    {
+        // lower redeliver timeout and claim interval
+        $connection = Connection::fromDsn(getenv('MESSENGER_REDIS_DSN'), [], $this->redis);
+
+        $connection->cleanup();
+        $connection->setup();
+
+        $body = '{"message": "Hi"}';
+        $headers = ['type' => DummyMessage::class];
+
+        // Add two messages
+        $connection->add($body, $headers);
+
+        // Read first message with other consumer
+        $this->redis->xreadgroup(
+            'symfony',
```

Comment:
read group name from `$connection->getGroup()` as the DSN can be configured different to a different stream when testing locally.